packages/fleather/lib/src/widgets/editor.dart

```diff
@@ -1265,11 +1265,6 @@ class RawEditorState extends EditorState
     performSpellCheck(widget.controller.plainTextEditingValue.text);
   }
 
-  bool _shouldShowSelectionHandles() {
```

Comment:
What was the purpose of this function that is removed now?

Reply:
It was used to show or hide the handles depending on whether the selection was collapsed or not.
Now the show/hide of the handles is entirely managed by `EditorTextSelectionOverlay` allowing for platform specific behavior